src/render/processor.rs

```diff
@@ -54,6 +55,10 @@ pub trait AudioProcessor: Send {
         params: AudioParamValues,
         scope: &RenderScope,
     ) -> bool;
+
+    fn event_emitter(&mut self) -> Option<&mut dyn EventEmitter> {
```

Comment:
This is now part of the public API, which I think we should avoid

Reply:
yup, I made few things public just to make it work but of course this is not good, with a common `AudioNodeId`I think this particular issue would disappear